bitcoin/src/crypto/key.rs

```diff
@@ -22,64 +22,6 @@ use crate::prelude::*;
 use crate::taproot::{TapNodeHash, TapTweakHash};
 use crate::{base58, io};
 
-/// A key-related error.
```

Comment:
random hint on this commit: for rebase hell, it's probably smarter to do the move commit first to simplify rebases :)

Reply:
You mean to simplify rebases where I want to change the PR, right? (As opposed to rebasing to fix merge conflicts.) If so, good point, will keep in mind. Thanks